test-data/unit/pythoneval.test

```diff
@@ -1590,7 +1566,6 @@ _testNoPython3StubAvailable.py:3: note: (or run "mypy --install-types" to instal
 
 
 [case testTypingOrderedDictAlias]
-# flags: --python-version 3.7
 from typing import OrderedDict
 x: OrderedDict[str, int] = OrderedDict({})
 reveal_type(x)
```

Comment:
This will probably fail. The output line will need to be changed from `4` to `3`.
_Can't do a suggestion for that line, unfortunately._